packages/amplication-client/src/Topics/TopicForm.tsx

```diff
@@ -69,7 +69,7 @@ const TopicForm = ({ onSubmit, defaultValues }: Props) => {
           label="Display Name"
```

Comment:
replace this intance of <TopicNameField/> with <DisplayNameField/>

Reply:
The issue is not here. This is a correct label.